exercises/practice/intergalactic-transmission/.docs/introduction.md

```diff
@@ -0,0 +1,10 @@
+# Introduction
+
+Trillions upon trillions of messages are exchanged between Earth and the neighboring galaxies every millisecond. 
+But transmission over such long distances faces many obstacles.
+Pesky solar flares, worm holes, temporal distortions, stray forces, heck even the flap of a space butterfly wing can cause a random bit to flip during transmission.
+The difference between an "o" and a "k", a "[" and "{" or a cowboy emoji and a clown emoji can have huge ramifications, from missed orders to Galactic Stockmarket crashes.
+It is important for the receiver to know when the message has been corrupted so that the receiver can ask for the message to sent again.
+
+One day, an important message comes through.
+People have just heard about the legend about a way of transmitting in such a way that the receiver can tell when bit flipped ... the parity bit ...
```

Comment:
I'm not sure I know a better phrasing, but this sentence reads a little weird to me. But we can leave it for now and have others comment on it when we do a prob-specs PR.